api/app/src/app.ts

```diff
@@ -13,6 +13,7 @@ app.use(helmet()); // needs to come before any route declaration, including cors
 app.use(express.json({ limit: "2mb" }));
 app.use(express.urlencoded({ extended: false, limit: "2mb" }));
 app.use(cors());
+app.options("*", cors());
```

Comment:
is this still necessary after the CDK patch?

Reply:
The main change is on API GW. Left the change at the API's app.ts because of [this](https://www.npmjs.com/package/cors#enabling-cors-pre-flight).

It was not failing without it, but also it was not failing local without allowing all headers while setting up `cors()`.

I could test without it, but needs a whole cycle to push code to staging to test it. Reckoned it was not harmful and makes sense to have it based on the cors library docs.

Reply:
What do you think?

Reply:
Removed the code on `app.ts` and sent it to staging: it still works.

So this PRs only adds the change to allow all headers on API GW for CORS.